### ISSUE TITLE
Flame.audio.play crashes `'Future<dynamic>' is not a subtype of type 'Future<int>'`

### ISSUE THREAD

**Issue by gtsipina** [luanpotter/flame#11]
In my project with Dart2, when I use `Flame.audio.play`, it prints following exception to the console.

```
[VERBOSE-2:dart_error.cc(16)] Unhandled exception:
type 'Future<dynamic>' is not a subtype of type 'Future<int>'
#0      AudioPlayer.play (package:audioplayers/audioplayer.dart:28:16)
#1      Audio.play (package:flame/audio.dart:47:37)
<asynchronous suspension>
```

**Comment by luanpotter** [comment]
Thanks for your feedback! What version of Flame are you using? This was fixed in audioplayers 0.5.1, which should be included in the latest version of flame. Could you verify that, please?

**Comment by gtsipina** [comment]
Thank you for answer. Version of Flame: ^0.8.4. I didn't find newer version...

**Comment by luanpotter** [comment]
I just released 0.9.0 with the new audioplayer version! See if that helps. Otherwise check in your pubspec.lock what version of audioplayers is being resolved :)

Indeed the latest released version of flame potentially allowed for an older version of audioplayers that didn't compatibility for Dart 2.

**Comment by gtsipina** [comment]
Thank you! The new version works!
